basics/a27.go

```diff
@@ -0,0 +1,37 @@
+// Understanding basic variable data types
+
+package main
+
+import "fmt"
+
+func main() {
+	var a = "/home"
+	fmt.Printf("Data type of variable a is %T\n", a)
+
+	var aa = 'a'
+	fmt.Printf("Data type of variable aa is %T\n", aa)
```

Comment:
your fmt.Printf doesnt have quotes.

`mt.Printf("Data type of variable 'aa' is %T\n", aa)`

Reply:
updated the code.